src/styles/stars.scss

```diff
@@ -0,0 +1,29 @@
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 16px;
+  width: 200px;
+}
+
+.card__stars-list {
+  display: flex;
+}
+
+.card__star:not(:last-child) {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  padding-right: 4px;
+}
+
+.card__star:last-child {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
```

Comment:
Similar to the previous comment, the border-radius value of 0.5px might be too small to notice. Consider increasing it if you want a more visible rounded effect.